internal/setup/yaml.go

```diff
@@ -25,6 +25,17 @@ type yamlRelease struct {
 	Format   string                 `yaml:"format"`
 	Archives map[string]yamlArchive `yaml:"archives"`
 	PubKeys  map[string]yamlPubKey  `yaml:"public-keys"`
+	// "v2-archives" defines the archives, same as "archives". It is added to
```

Comment:
I think this comment is a bit too lengthy, let's try to simplify it. What about something like?
```go
	// "v2-archives" is used for backwards compatibility with Chisel <= 1.0.0
	// since it will be ignored. In new versions it will be parsed with the new
	// fields that break said compatibility, like "pro" archives.
	//
	// Note: "archives" and "v2-archives" are merged together while parsing.
```
I was going to comment to move the TODO outside of the public documentation which I think is something you should always do. However, I think that in this case is better to omit the TODO altogether. We are not sure when we are going to remove this field and we do not want to make implicit promises in the code tied to compatibility with a given version.

Reply:
Thanks! Copied your comment with a few words changed.